cmd/kraft/x/exp.go

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: BSD-3-Clause
+// Copyright (c) 2023, Unikraft GmbH and The KraftKit Authors.
+// Licensed under the BSD-3-Clause License (the "License").
+// You may not use this file except in compliance with the License.
+package x
+
+import (
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+
+	"kraftkit.sh/cmdfactory"
+
+	"kraftkit.sh/cmd/kraft/x/probe"
+)
+
+type Exp struct{}
+
+func New() *cobra.Command {
+	cmd, err := cmdfactory.New(&Exp{}, cobra.Command{
+		Short: "Experimental commands",
+		Use:   "x [SUBCOMMAND]",
+		Long: heredoc.Doc(`
+		Run experimental commands.
+		`),
```

Comment:
Can you remove the `Long` attribute (since it's the same as the `Short`) and add `Hidden: true`